core/src/utils/native/capacitor.ts

```diff
@@ -0,0 +1,13 @@
+import type { CapacitorGlobal } from '@capacitor/core';
+
+let capacitor: CapacitorGlobal | undefined;
+
+export const getCapacitor = () => {
+  if (capacitor !== undefined) {
+    return capacitor;
+  }
+  if (window !== undefined) {
```

Comment:
We should use the `win` util otherwise this will throw in SSR: https://github.com/ionic-team/ionic-framework/blob/82c568b8c8e1e9934e1928452aa5216619290e7b/core/src/utils/browser/index.ts#L23

Reply:
I still think this feedback is useful and we should use the `win` util to avoid any unexpected issues in the future. Addressed here: https://github.com/ionic-team/ionic-framework/pull/27755/commits/07db5fea3dca832c952aa8e1a747dd6e51f3be60